shipments_tracker/app/models/shipment.rb

```diff
@@ -5,11 +5,9 @@ class Shipment < ApplicationRecord
   # Validación: El campo status no puede estar vacío y solo puede contener los valore especificados
   validates :status, presence: true, inclusion: { in: %w(REGISTERED IN_TRANSIT DELAYED LOST STOLEN DELIVERED) }
 
-  # Validación: El campo tracking_history no puede estar vacío y debe tener al menos un elemento (longitud mínima de 1)
-  validates :tracking_history, presence: true, length: { minimum: 1 }
```

Comment:
this same change is apply for victor in this [PR](https://github.com/BrightCoders-Institute/proyecto-final-ror-team1/pull/25/files#diff-65021091b0c955d78481490438cb3351dbba5144846c4b5b4c145b9b1e11d582L9)

Reply:
ni entiendo bien, ¿que debo corregir ?

Reply:
Done!